File: ejercicio py/ejercicio2.py
nota1 = float(input("ingrese su primer nota:"))
nota2 = float(input("ingrese su seguda nota:"))
nota3 = float(input("ingrese su tercera nota:"))
prom =(nota1 + nota2 + nota3)/3

if prom >= 3.5 and prom <= 4.0:
    print(f"se obtuvo un promedio de {prom} esta: aprobado")
elif prom >= 3.5:
    print(f"se obtuvo un promedio de {prom} eres: sobresaliente ")
else:
    print(f"se obtuvo un promedio de {prom} esta: desaprobado")

File: ejercicio py/ejercicio2.php
<?php
//Determinar si un aprendiz aprobo la materia dadas 3 notas las materias 
//se aprueban con 3.5, el sistema debera mostrar un mensaje de aprobo, 
//reprobo y el promedio de la materia 


$nota1 = readline("Ingrese la nota 1 ");
$nota2 = readline("Ingrese la nota 2 ");
$nota3 = readline("Ingrese la nota 3 ");

$promedio = ($nota1 + $nota2 + $nota3) / 3;


if ($promedio >= 3.5) {
    echo "Aprobado con un promedio de" . $promedio;
} elseif ($promedio < 3.5) {
    echo "Reprobado con un promedio de " . $promedio;
} else {
    echo "Valores invalidos";
}

?>